code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -119,19 +119,19 @@
 	wardenhead["warden beret, corporate"] = /obj/item/clothing/head/beret/security/warden/corp
 	gear_tweaks += new /datum/gear_tweak/path(wardenhead)
 
-/datum/gear/head/hos
-	display_name = "headwear, security (head of security)"
-	description = "A selection of head of security headwear."
+/datum/gear/head/cos
+	display_name = "headwear, security (Chief of Security)"
+	description = "A selection of Chief of Security headwear."
```

Comment:
```suggestion
	description = "A selection of chief of security headwear."
```

Reply:
Loadout, it's capitalized there, so it stays that way.